Source/WebCore/platform/graphics/gstreamer/mse/AppendPipeline.cpp

```diff
@@ -1196,9 +1196,7 @@ void AppendPipeline::connectDemuxerSrcPadToAppsink(GstPad* demuxerSrcPad)
     ASSERT(WTF::isMainThread());
     GST_DEBUG("Connecting to appsink");
 
-    const String& type = m_sourceBufferPrivate->type().containerType();
-    if (type.endsWith("webm"))
```

Comment:
I preffer to change it to:
if (type.endsWith("webm") || type.endsWith("fmp4"))

Reply:
I can do that, but in other place we already do:
    const String& type = m_sourceBufferPrivate->type().containerType();
    if (type.endsWith("mp4"))
        m_demux = gst_element_factory_make("qtdemux", nullptr);
    else if (type.endsWith("webm"))
        m_demux = gst_element_factory_make("matroskademux", nullptr);
    else
        ASSERT_NOT_REACHED();


So I'm 100% sure we won't have any other type of content there.